projects/packages/forms/src/blocks/shared/hooks/use-form-style-outline-classes-and-styles.js

```diff
@@ -0,0 +1,163 @@
+import {
+	store as blockEditorStore,
+	__experimentalGetBorderClassesAndStyles as getBorderClassesAndStyles, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+	__experimentalGetColorClassesAndStyles as getColorClassesAndStyles, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+} from '@wordpress/block-editor';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { useMemo } from '@wordpress/element';
+import deepmerge from 'deepmerge';
+import { isPlainObject, isNumber } from 'lodash';
```

Comment:
This might be a bit nitpicky, but since we're already using `lodash` in Jetpack, could we use lodash's `merge` rather than adding another dependency? Or is `deepmerge` needed?

Reply:
Not at all. Good call. I was hacking here.